graphstorm-processing/docker/README.md

```diff
@@ -12,34 +12,48 @@ with Amazon SageMaker see docs/source/usage/distributed-processing-setup.rst.
 ## Building the image
 
 To build the image you will run `bash build_gsprocessing_image.sh`
-script that has one required parameter, `--target` that can take
-one of two values, `prod` and `test` that determine whether we
-include the source and tests on the image (when `test` is used),
-or just install the libary on the image (when `prod` is used).
```

Comment:
Why do we delete the target description here? Although we currently enable the prod as the default one. We still provide option to let user choose the target for prod and test.

Reply:
The main text here used to explain the only _required_ argument that was previously target (when this was written). The only required argument now is environment.

We add the description of target as one of the arguments below.